website/content/en/preview/getting-started/migrating-from-cas/scripts/step05-tag-subnets.sh

```diff
@@ -1,6 +1,6 @@
-for NODEGROUP in $(aws eks list-nodegroups --cluster-name ${CLUSTER_NAME} \
-    --query 'nodegroups' --output text); do aws ec2 create-tags \
+for NODEGROUP in $(aws eks list-nodegroups --cluster-name "${CLUSTER_NAME}" --query 'nodegroups' --output text); do
+    aws ec2 create-tags \
         --tags "Key=karpenter.sh/discovery,Value=${CLUSTER_NAME}" \
-        --resources $(aws eks describe-nodegroup --cluster-name ${CLUSTER_NAME} \
-        --nodegroup-name $NODEGROUP --query 'nodegroup.subnets' --output text )
+        --resources "$(aws eks describe-nodegroup --cluster-name "${CLUSTER_NAME}" \
```

Comment:
Does quoting work inside of quoting here?

Reply:
Yes, it's best practice to always quote strings in bash.